test/graphgym/test_graphgym.py

```diff
@@ -31,8 +31,16 @@
 )
 from torch_geometric.testing import withPackage
 
+try:
```

Comment:
actually, the minimal installation test fail once I remove the try block. Maybe would have to remove the `graphgym` module from minimal tests for this?

Reply:
Not sure I understand. Why can't we do `import pytorch_lightning` inside the `test_*` function?

Reply:
I think it is because pytest goes to every test module and since PL import is defined globally, it will import PL without executing `@withPackage('pytorch_lightning')`. This results in import error and it fails the minimal installation test.